sdk/schemaregistry/schema-registry/review/schema-registry.api.md

```diff
@@ -21,43 +21,44 @@ export interface RegisterSchemaOptions extends OperationOptions {
 }
 
 // @public
-export interface Schema extends SchemaProperties {
-    schemaDefinition: string;
+export interface Schema {
+    definition: string;
+    properties: SchemaProperties;
 }
 
 // @public
 export interface SchemaDescription {
+    definition: string;
     format: string;
     groupName: string;
     name: string;
```

Comment:
Should this be schemaName?

Reply:
Unlike the other languages, these properties are under `SchemaDescription` so `SchemaDescription.name` and `SchemDescription.groupName` are clear enough to convey the meaning that the former is a schema name and the latter is for the group name where the schema belongs to. Also, there is no schema prefix for any other field in it so adding it to the name field only will make the naming inconsistent. I can change it though if you feel strongly about it.